rustpad-server/src/lib.rs

```diff
@@ -208,19 +209,25 @@ async fn cleaner(state: ServerState, expiry_days: u32) {
 }
 
 const PERSIST_INTERVAL: Duration = Duration::from_secs(3);
+const PERSIST_INTERVAL_JITTER: Duration = Duration::from_secs(1);
 
 /// Persists changed documents after a fixed time interval.
 async fn persister(id: String, rustpad: Arc<Rustpad>, db: Database) {
     let mut last_revision = 0;
     while !rustpad.killed() {
-        time::sleep(PERSIST_INTERVAL).await;
+        let interval = PERSIST_INTERVAL
+            + Duration::from_millis(
+                rand::thread_rng().gen_range(0..=PERSIST_INTERVAL_JITTER.as_millis() as u64),
+            );
```

Comment:
Thanks, since `Duration` implements `SampleUniform` I believe you can write this as `gen_range(Duration::ZERO..=PERSIST_INTERVAL_JITTER)`